### PR TITLE
New methods page() offset() clause()

### PR DESCRIPTION
**Pull request by damianoporta**

### CODE REVIEW

File: src/Query.php [hunk 1]
```diff
@@ -109,6 +110,56 @@ public function limit($limit)
...
+        }
+        $offset = ($num - 1) * $limit;
+        if (PHP_INT_MAX <= $offset) {
+            $offset = PHP_INT_MAX;
+        }
+        $this->offset((int)$offset);
+        return $this;
+    }    
+    
+    public function clause($name)
```
Comment (lorenzo) [inline src/Query.php]: Needs a dock block

**Comment by lorenzo** [comment]
Can you add tests for the new methods?

**Comment by damianoporta** [comment]
@lorenzo ok

File: src/Query.php (hunk 1 above)
Comment (damianoporta) [inline src/Query.php]: Done

**Comment by markstory** [comment]
Looks good to me.

**Pull request merged**